.rubocop.yml

```diff
@@ -19,8 +19,17 @@ Layout/IndentFirstHashElement:
 Layout/IndentHeredoc:
   Enabled: false
 
+Metrics/BlockLength:
+  Max: 40
```

Comment:
Note that we can make this much more restrictive after excluding the test directory. It moves from 509 to 40.